TileSystem/Interfaces/Creation/ICreateTiles.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+using TileSystem.Interfaces.Base;
+
+namespace TileSystem.Interfaces.Creation
+{
+	/// <summary>
+	/// ICreateTiles is used by the level to create tiles
+	/// using a normal creational pattern
+	/// 
+	/// The area, x, y will be where the tile is created
+	/// </summary>
+	public interface ICreateTiles
+	{
+		event EventHandler<TileCreatedArgs> TileCreated;
+		ITile CreateTile(IArea area, int x, int y, string type, string variation, params object[] properties);
```

Comment:
it seems to be the only interface that use directly **[x, y]**

Is it possible to create an Interface **ILocation** to abtract that and not enforce this 2d array representation.
Allowing nodes, 3d or whatever other crazy idea and let that to the implementation